src/patch-builtins.js

```diff
@@ -55,7 +55,11 @@ let nodeMixin = {
   },
 
   get isConnected() {
-    return document.documentElement.contains(this)
+    let node = this;
```

Comment:
Can we fast path this instead to use 

`node.ownerDocument.documentElement.contains(this)` ?